docker-compose.yaml

```diff
@@ -1,12 +1,12 @@
 x-default: &node
-  image: ${CCAO_REGISTRY_URL}/service-spark-iasworld:latest
   # Corresponds to the user set in the Dockerfile and the shiny-server
   # user on the server for proper write perms to mounted directories
-  user: 1003:0
+  user: "${UID:-1003}:0"
   build:
     context: .
     dockerfile: Dockerfile
-  restart: unless-stopped
+    args:
+      - USER_ID=${UID:-1003}
```

Comment:
These UID args pass into the container during build and run time. I'm not positive that it's required to specify the build time user in the Dockerfile, but it can't hurt either.

Reply:
Fixed in the most recent revisions!